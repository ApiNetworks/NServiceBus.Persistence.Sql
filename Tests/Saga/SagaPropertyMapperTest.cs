using NUnit.Framework;

[TestFixture]
public class SagaPropertyMapperTest
{
    [Test]
    public void TryGetBaseSagaType()
    {
        SagaPropertyMapper.ExtractProperty<SagaPropertyMapperTest>(test => test.Property);
    }

    public object Property { get; set; }
}