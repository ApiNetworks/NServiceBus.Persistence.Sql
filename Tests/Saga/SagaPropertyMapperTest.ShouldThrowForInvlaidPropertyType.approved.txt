The type System.Object is not valid to be used as a mapped message or as a unique property. The valid types are System.String, System.Char, System.Int32, System.Int64, System.Int16, System.UInt32, System.UInt64, System.UInt16, System.Guid, System.Single, System.DateTime, System.DateTimeOffset, System.TimeSpan